adserver/templates/adserver/includes/widgets/advertisement-crop-resize-modal.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n %}
+
+<!-- Modal -->
+<div class="modal fade" id="modal" tabindex="-1" aria-labelledby="modalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="modalLabel">{% trans 'Crop and resize image' %}</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <p>{% trans 'Your image is not the right size for the ad format. You will need to crop and resize the image.' %}</p>
```

Comment:
Should we mention the size here, or will it be displayed in the UI somewhere else?